data/pages-content/social-event.md

```diff
@@ -1,65 +1,57 @@
 ---
 title: Social Event
 subtitle:
-  Come to the EuroPython 2022 social event on Thursday evening!
+  Join us at the EuroPython 2023 social event on Thursday evening! Let's eat, chat and laugh!
+social_card: ep2023-social-event-sc.png
 ---
-## EuroPython 2022 Social Event ##
-After the keynotes and talks on Thursday, July 14, we've organized a social event for you at the conference venue,
-the Dublin Convention Centre (CCD). Join us for an evening party with food, drinks
-and live traditional Irish music!
+<div style={{marginBottom: 15}}>
+  <img src="/img/ep2023-social-event.png" />
+</div>
+
+There are venues that are nice. There are venues that are interesting. And then there are venues that have stories - such as the home of our social event venue, a 200-year-old deconsecrated former church [Sacre Coeur](https://www.sacrecoeur.cz/en). Join us on the evening of 20 July at for an evening gathering with food, chats and live folk music!
```

Comment:
I have just added on some punctuation 🙈 , feel free to ignore this comments as cause it doesn't change the content at all 😊 
> There are nice venues, there are interesting venues, and then there are venues with stories to tell—like Sacre Coeur, the home of our social event venue, a 200-year-old deconsecrated former church. Join us on the evening of July 20th for a gathering filled with food, conversations, and live folk music!

Reply:
I cannot ignore the comment when it is such a great comment! Suggestions applied with thanks!